src/app/components/drawer/SavedSearches/SavedSearchActions.js

```diff
@@ -243,14 +297,14 @@ const ProjectActions = ({
           <FormattedMessage
             defaultMessage="Delete list"
             description="'Delete' here is an infinitive verb"
-            id="projectsComponent.deleteType"
+            id="projectActions.deleteType"
           />
         }
         title={
           <FormattedMessage
             defaultMessage="Delete list?"
             description="'Delete' here is an infinitive verb"
-            id="projectsComponent.deleteTitleType"
+            id="projectActions.deleteTitleType"
```

Comment:
Looks like there's still some ID changes in this file `projectsComponent` changed `projectActions`

Reply:
Done

Reply:
Done